manifest.json

```diff
@@ -6,7 +6,7 @@
     "url": "http://strut.io",
     "license": "AGPL-3.0",
     "requirements": {
-        "yunohost": ">= 2.7.2"
+        "yunohost": ">= 3.2.0"
```

Comment:
You use ynh_handle_getopts_args in _common.h, the minimum version for this is 3.2.2: https://github.com/YunoHost/yunohost/blob/873bf4ae370cf785c66db0028ca1ab5bc06340c0/data/helpers.d/getopts#L47

Reply:
Do not hesitate to fix directly, no need to ask for that